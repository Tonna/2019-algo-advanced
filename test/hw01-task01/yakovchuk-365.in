25
1 1 1 1 4 3 4 6 8 2 1 3 12 1 1 1 14 9 4 12 10 1 18 6
4
24 5
7 6
17 1
1 3
