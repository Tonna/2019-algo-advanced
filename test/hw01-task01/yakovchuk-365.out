29
11
17
3
